Sources/Slacket/Kitura/ServerConfig.swift

```diff
@@ -24,7 +24,7 @@ extension ServerConfig {
     }
     
     var host: String {
-        return "slacket.link"
+        return "localhost"
```

Comment:
That will break deployment. We would need something like that:
```
#if os(OSX)
return "localhost"
#else
return "slacket.link"
#endif
```

besides you need to remember that localhost config runs on a 8090 port where linux on 80